recipe/meta.yaml

```diff
@@ -1,36 +1,46 @@
 {% set name = "torchmetrics" %}
-{% set version = "0.11.1" %}
+{% set version = "0.11.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: de2e9feb3316f798ab08b318302ff04e764f47e691f0847f780044279fa176ca
+  # Update an url to https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # when the package will be available on PyPI.
+  url: https://github.com/Lightning-AI/metrics/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1c48fea9bc466cd618af1ce09134aa86829a4c8883305982710250df0058e711
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
 
 requirements:
   host:
     - pip
-    - python >=3.6
-  run:
-    - packaging
+    - python
     - setuptools
-    - python >=3.7
+    - wheel
+  run:
+    - python
+    - numpy >=1.17.2
```

Comment:
we miss here in host the numpy dependency.  Otherwise the pinning will fail to specify upper bounds proper.

Reply:
Done!